### PR TITLE
feat: add auto-assign to themes issues and PRs

### PR DESCRIPTION
**Pull request by afonsojramos**
@williamckha initially I had Fluent here, but I'll remove it in another PR.

There are a lot of themes that I do not know the owners yet however. If anyone could help here that would be awesome!

### CODE REVIEW

File: .github/workflows/auto-assign.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    steps:
+      - uses: Naturalclar/issue-action@v2.0.2
+        with:
+          title-or-body: "both"
+          parameters: |
+            [
+              {
+                "keywords": ["BurntSienna"],
+                "labels": ["🙋 help wanted"],
+                "assignees": ["username"]
```
Comment (morpheusthewhite) [inline .github/workflows/auto-assign.yml:20]: https://github.com/pjaspinski

File: .github/workflows/auto-assign.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              },
+              {
+                "keywords": ["Default"],
+                "labels": ["🙋 help wanted"],
+                "assignees": ["username"]
```
Comment (morpheusthewhite) [inline .github/workflows/auto-assign.yml:25]: https://github.com/Blacksuan19

File: .github/workflows/auto-assign.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+              },
+              {
+                "keywords": ["Dreary"],
+                "labels": ["🙋 help wanted"],
+                "assignees": ["username"]
```
Comment (morpheusthewhite) [inline .github/workflows/auto-assign.yml:30]: https://github.com/CharlieS1103

File: .github/workflows/auto-assign.yml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+              },
+              {
+                "keywords": ["Dribbblish"],
+                "labels": ["🙋 help wanted"],
+                "assignees": ["khanhas"]
+              },
+              {
+                "keywords": ["Glaze"],
+                "labels": ["🙋 help wanted"],
+                "assignees": ["username"]
```
Comment (morpheusthewhite) [inline .github/workflows/auto-assign.yml:40]: https://github.com/CharlieS1103

File: .github/workflows/auto-assign.yml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+              },
+              {
+                "keywords": ["Onepunch"],
+                "labels": ["🙋 help wanted"],
+                "assignees": ["username"]
```
Comment (morpheusthewhite) [inline .github/workflows/auto-assign.yml:45]: https://github.com/okarin001

File: .github/workflows/auto-assign.yml [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+              },
+              {
+                "keywords": ["Sleek"],
+                "labels": ["🙋 help wanted"],
+                "assignees": ["username"]
```
Comment (morpheusthewhite) [inline .github/workflows/auto-assign.yml:50]: https://github.com/harbassan

File: .github/workflows/auto-assign.yml [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+              },
+              {
+                "keywords": ["Turntable"],
+                "labels": ["🙋 help wanted"],
+                "assignees": ["username"]
```
Comment (morpheusthewhite) [inline .github/workflows/auto-assign.yml:55]: https://github.com/grasonchan

File: .github/workflows/auto-assign.yml [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+              },
+              {
+                "keywords": ["Ziro"],
+                "labels": ["🙋 help wanted"],
+                "assignees": ["username"]
```
Comment (morpheusthewhite) [inline .github/workflows/auto-assign.yml:60]: https://github.com/schnensch0

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]

**Comment by afonsojramos** [comment]
Thank you @morpheusthewhite! I'll also add all of those as contributors to this repo!

**Comment by afonsojramos** [comment]
Okay, it seems that it is working for everyone that is already part of the repo!

**Comment by afonsojramos** [comment]
Please accept the invites to join the Spicetify org and themes repository as soon as you can! 
@pjaspinski @Blacksuan19 @okarin001 @harbassan @grasonchan @schnensch0

**Review by morpheusthewhite — APPROVED** [review]

**Pull request merged**